app/javascript/components/ArtistWorks.js

```diff
@@ -0,0 +1,116 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+/**
+  * @prop buyer: buyer currently logged in
+  * @prop artist: artist associated with works
+  */
+class ArtistWorks extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      works: [],
+      comment: "",
+      errors: [],
+      componentDidMount: false
+    }
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount = () => {
+    const artist_id = this.props.artist.id;
+    const works_route = APIRoutes.artists.works(0);
+    Requester.get(works_route).then(
+      response => {
+        this.setState({
+          works: response,
+          componentDidMount: true
+        });
+      },
+      response => {
+        console.err(response);
+      }
+    );
+  }
+
+  handleChange(event) {
+    this.setState({ comment: event.target.value });
+  }
+
+  checkErrors() {
+    let errors = [];
+    if (!this.state.comment) {
+      errors.push("This field cannot be empty.");
+    }
+    if (!this.props.buyer) {
+      errors.push("You must be logged in to request a commission.");
+    }
+    this.setState({ errors: errors });
+  }
+
+  handleSubmit() {
+    this.checkErrors();
+    if (!this.state.errors) {
```

Comment:
Nothing happens when clicking the create button, problem is that javascript recognizes an empty list as true